### ISSUE TITLE
Values not down to zero end of the day

### ISSUE THREAD

**Issue by steenderen** [home-assistant/home-assistant.github.io#19774]
### Feedback

Hi, first of all, thanks very much for this integration!

I noticed that values are not going down to zero when all sun is gone. Internal power stays at around 60wat.

Besides the life time energy is going back to 0.

Any idea if anything strange is going on at my side? Or if something can be done?

### URL

https://www.home-assistant.io/integrations/growatt_server/

### Version

2021.10.4

**Comment by muppet3000** [comment]
Hi @steenderen 

Sorry that some of the values aren't performing as you would expect. Unfortunately this isn't something we're in control of as part of the integration. We just pull the values directly from the Growatt Server API (the one that the ShineApp on your phone uses). 
If your values aren't behaving as you'd expect on that then you my have to take it up directly with Growatt.

**Comment by steenderen** [comment]
OK, sounds logical, but I noticed that values in the Growatt app seem OK, while in Home Assistant they are not. So somehow they are not in sync. I got the feeling that after a certain time or something Home Assistant is not connecting to the API anymore for whatever reason.

**Comment by steenderen** [comment]
Some questions:
1. Can I somewhere see  logs or something of the API checks? If I look at this moment, than it looks like values are not updates since 18:13.
2. Is it possible to force an API check?

**Comment by steenderen** [comment]
OK, I found something to make it work for me (but still can't explain what is actually happening).

I've found there are two types of sensors available from this integration, the ones starting with sensor.fnh2b11xxx and the ones starting with the username of my account in it. The latter seem to work fine, and I use now the one with total_lifetime_energy_output in the name for the Energy dashboard. This one just adds up all power production in kWh, as you would expect.

The ones starting with sensor.fnh2b11xxx are still acting weird. Don't know if this is the integration causing it, or the API from Growatt. Every night around 00:00 the value of e.g. the sensor ending with lifetime_energy_output is going down to 0. And every morning I see the totale value coming back again. But going back to 0 causes the energy dashboord to show a total production of you lifetime_energy_output, which is an enormous spike.

Very strange, but as said, my solution by using the other sensor is working fine.

**Comment by muppet3000** [comment]
@steenderen FYI, this should have been logged over in the https://github.com/home-assistant/core/issues section as it's an issue with the integration rather than the UI (I've only just realised this).
Anyway, I believe this issue is actually a duplicate of this ticket: https://github.com/home-assistant/core/issues/56500
which only seems to affect certain types of systems and is really a problem with the Growatt server data itself. However we have discussed adding a potential workaround (which must be what Growatt decided to implement in the ShineApp).
